DigitalLearningSolutions.Data/Helpers/LinkedFieldHelper.cs

```diff
@@ -48,7 +48,7 @@ ICentreRegistrationPromptsService centreRegistrationPromptsService
             {
                 var oldJobGroup = jobGroupsDataService.GetJobGroupName(oldAnswers.JobGroupId);
                 var newJobGroup = jobGroupsDataService.GetJobGroupName(newAnswers.JobGroupId);
-                changedLinkedFieldsWithAnswers.Add(new LinkedFieldChange(4, "Job Group", oldJobGroup, newJobGroup));
+                changedLinkedFieldsWithAnswers.Add(new LinkedFieldChange(4, "Job group", oldJobGroup, newJobGroup));

```

Comment:
@SteveJacksonSoft I tested the synchronisation process carefully and it still works fine. Except I spotted two bugs - the first is that groups based on job groups that were prefixed with "Job group" weren't being synchronised because of this capitalisation inconsistency. I fixed it to how I add the prefix upon group generation, which is also how we spell it on the My Account details page, but let me know if we should treat it case insensitively

Reply:
Oh good spot. Let's be case-insensitive when checking - there's no risk of incorrectly matching other strings.

Reply:
@SteveJacksonSoft Do we care about case sensitivity with the rest of the group name? Can you imagine a case where there might be two answers to a registration prompt that are identical except for capitalisation?

Reply:
An apt question. We're going for case-insensitive uniqueness in https://softwiretech.atlassian.net/browse/HEEDLS-596 so this shouldn't be possible. Let's treat two answers that differ by case alone as the same thing.

Reply:
Great, I will make that change on the 805 branch